habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -212,20 +212,29 @@ def tile_images(render_obs_images: List[np.ndarray]) -> np.ndarray:
     return final_im
 
 
-def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
+def observations_to_image(observation: Dict, info: Dict, index=0) -> np.ndarray:
     r"""Generate image of single frame from observation and info
     returned from a single environment step().
 
     Args:
         observation: observation returned from an environment step().
         info: info returned from an environment step().
+        index: index of the frame being rendered.
 
     Returns:
         generated image of a single frame.
     """
     render_obs_images: List[np.ndarray] = []
+    # TODO(ZXZ): for frames storage
+    robot_names = {
+        'agent_0': 'SpotRobot',
+        'agent_1': 'DJIDrone',
+        'agent_2': 'FetchRobot',
+        'agent_3': 'StretchRobot',
+    }
```

Comment:
Can we make it a general script so that each robot names are retrieved from the yaml config instead of hard coding here?

Reply:
Solved:)